server/src/coreV2/user/user.auto-resolver-opts.ts

```diff
@@ -0,0 +1,42 @@
+import { SortDirection } from '@ptc-org/nestjs-query-core';
+import {
+  AutoResolverOpts,
+  ReadResolverOpts,
+  PagingStrategies,
+} from '@ptc-org/nestjs-query-graphql';
+
+import { JwtAuthGuard } from 'src/guards/jwt.auth.guard';
+
+import { User } from './user.entity';
+
+import { CreateUserInput } from './dtos/create-user.input';
+import { UpdateUserInput } from './dtos/update-user.input';
+
+export const userAutoResolverOpts: AutoResolverOpts<
+  any,
+  any,
+  unknown,
+  unknown,
+  ReadResolverOpts<any>,
+  PagingStrategies
+>[] = [
+  {
+    EntityClass: User,
+    DTOClass: User,
+    CreateDTOClass: CreateUserInput,
```

Comment:
@gitstart-twenty After reflexion, I don't think we need resolver for `create` and `update` based on what was done on Prisma side.
Actually `Authentication` will handle that by calling user service.
The `Authentication` module should be duplicate too to make it easier.